src/wallet/api/wallet2_api.h

```diff
@@ -1144,6 +1154,519 @@ struct Wallet
+// TODO : accept_func with wallet2::signed_tx_set
+    /**
+    * brief: loadTx - load pending transactions from a file
+    * param: signed_filename -
+    * outparam: ptx -
+    * param: accept_func - callback function to verify the transaction
+    * return: true if succeeded
+    */
+//    virtual bool loadTx(const std::string &signed_filename, PendingTransaction &ptx, std::function<bool(const signed_tx_set&)> accept_func) const = 0;
+// TODO : accept_func with wallet2::signed_tx_set
+    /**
+    * brief: parseTxFromStr - get transactions from encrypted signed transaction as hex string
+    * param: signed_tx_str -
+    * outparam: ptx -
+    * param: accept_func - callback function to verify the transaction
+    * return: true if succeeded
+    */
+//    virtual bool parseTxFromStr(const std::string &signed_tx_str, PendingTransaction &ptx, std::function<bool(const signed_tx_set &)> accept_func) const = 0;
+    /**
+    * brief: parseMultisigTxFromStr - get pending multisig transaction from encrypted unsigned multisig transaction as hex string
+    * param: multisig_tx_str -
+    * outparam: exported_txs -
+    * return: true if succeeded
+    * note: sets status error on fail
+    */
+    virtual bool parseMultisigTxFromStr(const std::string &multisig_tx_str, PendingTransaction &exported_txs) const = 0;
+// TODO : accept_func with wallet2::multisig_tx_set
+    /**
+    * brief: loadMultisigTxFromFile - load a multisig transaction set from a file
+    * param: multisig_tx_str -
+    * outparam: exported_txs -
+    * return: true if succeeded
+    */
+//    virtual bool loadMultisigTxFromFile(const std::string &filename, PendingTransaction &exported_txs, std::function<bool(const multisig_tx_set&)> accept_func) const = 0;
+// TODO : accept_func with wallet2::multisig_tx_set
+    /**
+    * brief: signMultisigTxFromFile - load a multisig transaction set from a file, sign it and store to a file
+    * param: filename - load from and store to file with this filename
+    * outparam: txids - transaction ids of signed transactions
+    * param: accept_func - callback function to verify the transaction
+    * return: true if succeeded
+    */
+//    virutal bool signMultisigTxFromFile(const std::string &filename, std::vector<crypto::hash> &txids, std::function<bool(const multisig_tx_set&)> accept_func) const = 0;
+    /**
+    * brief: getFeeMultiplier -
+    * param: priority -
+    * param: fee_algorithm -
+    * return: fee multiplier
+    */
+    virtual std::uint64_t getFeeMultiplier(std::uint32_t priority, int fee_algorithm) const = 0;
+    /**
+    * brief: getBaseFee -
+    * return: dynamic base fee estimate if using dynamic fee, else FEE_PER_KB
+    */
+    virtual std::uint64_t getBaseFee() const = 0;
+    /**
+    * brief: getMinRingSize - get the minimum allowed ring size
+    * return: min ring size
+    */
+    virtual std::uint64_t getMinRingSize() const = 0;
```

Comment:
Can go. We don't have the possibility to choose ringsize anymore for years, and well, soon we won't have any rings at all. Maximum that I see here is an info "ring size" in the new `WalletState`.

Reply:
Would require something like this: https://github.com/tobtoht/monero/commit/894adf09e7b87f9035562255d2a4c1198e338d4b. Can PR.